force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -55,10 +58,15 @@ public with sharing class FlowControlLoopsCollections {
      * @return The maximum of the two numbers, or null if either number is null.
      */
     public static Integer findMax(Integer num1, Integer num2) {
-        // Initialize largestNum as null
-        Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        if (num1 >= num2){ //checking for greater and equal to for num1
+            return num1; //return num1
+        }
+        else if (num2 > num1){ //checking for num2 greater than num1
+            return num2; //return num2
+        }
+        else {
+            return null; //for anyother case return null
```

Comment:
There is no other case here that would happen, you already checked >= and <, so this could never be reached. If you were concerned one of them is null, that check needs to be first, before the other comparisons.